html/browsers/windows/opener-cross-origin-manual.sub.html

```diff
@@ -0,0 +1,10 @@
+<ol>
+ <li><p>Clicking this link must navigate this page to example.com:
+ <a href=//www1.{{location[host]}}/html/browsers/windows/support-opener.html target=_blank>test</a>
+
+ <li><p>Clicking this link must open a new browsing context that is empty:
+ <a rel=noreferrer href=//www1.{{location[host]}}/html/browsers/windows/support-opener.html target=_blank>test</a>
+
+ <li><p>Clicking this link must navigate this page to example.com:
+ <a href=//www1.{{location[host]}}/html/browsers/windows/support-embed-opener.sub.html target=_blank>test</a>
```

Comment:
In Chrome, the navigation fails, but the `close()` doesn't. I'm not sure if I agree with that behavior.